contents/handbook/people/team-structure/app-small-teams.md

```diff
@@ -0,0 +1,41 @@
+---
+title: App Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: false
+---
+
+App Small Teams are particular types of Small Teams that own an area of the product.
+
+## What does owning an area of the product mean?
+
+The app small team is responsible for everything related to that area of the product including:
+
+1. Usage
+2. Quality
+3. Revenue
+
+## What actions should the app small teams be doing for their area?
+
+- Each quarter:
+  1. Create good quarterly OKRs
+  2. Create an ambitious vision for their area
+- During the quarter:
+  1. Maintain a prioritized roadmap to help them achieve their objectives
+  2. Speak to customers
+  3. Monitor relevant metrics including those covering Usage, Quality and Revenue
+  4. Triage and fix related bugs
+  5. Assist the support hero in answering related questions
+  6. Collaborate with marketing
+
+## What is the role of the team lead?
+
+The team lead should led the team to make sure the above happens. They should get the team fully involved rather that doing everything themselves.
+
+## What do the product managers do?
+
+With our engineering-led culture, the engineers on the app small team are responsible for the product. 
+
+We have a small number of product managers who support the app small teams in achieving their goals. This includes a wide range of activities such as helping with prioritization, setting up dashboards, competitors analysis, speaking to customers etc.
```

Comment:
Of these, setting up dashboards seems like the odd one out for me - given the teams priorities are clear, anyone on the team should be empowered to set up and maintain metrics.

Reply:
>  setting up dashboards seems like the odd one out for me - given the teams priorities are clear, anyone on the team should be empowered to set up and maintain metrics.

Yep, feels like, as with prioritization, product folk should be the local experts enabling everyone to do it.

Reply:
I agree with this "anyone on the team should be empowered to set up and maintain metrics" @macobo and was what I meant with that sentence. Any ideas how to frame it better?